package.json

```diff
@@ -1,19 +1,50 @@
 {
   "name": "sn-components-api",
-  "version": "1.2.8",
-  "main": "dist/dist.js",
-  "devDependencies": {
-    "@babel/cli": "^7.6.4",
-    "@babel/core": "^7.6.4",
-    "@babel/preset-env": "^7.6.3",
-    "grunt": "^1.0.4",
-    "grunt-babel": "^8.0.0",
-    "grunt-browserify": "^5.3.0",
-    "grunt-contrib-watch": "^1.1.0",
-    "grunt-newer": "^1.3.0"
-  },
+  "version": "2.0.0",
   "repository": {
     "type": "git",
     "url": "git://github.com/sn-extensions/components-api.git"
+  },
+  "main": "dist/dist.js",
+  "types": "dist/@types",
+  "scripts": {
+    "build:docs": "typedoc",
+    "build:types": "tsc",
+    "bundle": "yon run lint:fix && webpack --config webpack.prod.js && yon run build:types",
```

Comment:
typo? `yon` instead of `yorn`?

Reply:
I was using `yarn-or-npm` here (`yon` is just the shorthand version). I'll just use `yarn` instead.